website/docs/intelligence-kit/overview-intel.md

```diff
@@ -0,0 +1,38 @@
+---
+id: overview-intel
+title: Intelligence Kit Overview
+---
+
+![DSCF8544](https://github.com/farm-ng/amiga-dev-kit/assets/64480560/3cfbeb81-5feb-4dcf-8fbd-2fea69f1ca98)
+
+[Buy it on Farm-ng Shop](https://farm-ng.com/products/amiga-brain-kit)
+
+The Amiga Intelligence Kit is a comprehensive system designed for developers and researchers seeking to add to their Amiga's capabilities for exploration, data collection and more.
+
+## What's Included
+
+### Brain
+
```

Comment:
This file is not going to pass pre-commit formatting checks once you merge main into your branch

Reply:
Oh because of the line maximum. I will make those edits.